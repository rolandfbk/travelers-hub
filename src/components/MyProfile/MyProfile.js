import React from 'react';
import RocketsReserved from '../Rockets/RocketsReserved';
import MissionReservedList from '../MissionReservedList';

const MyProfile = () => (
  <div className="my-profile">
    <div className="my-profile-align">
      <MissionReservedList />
      <RocketsReserved />
    </div>
  </div>
);

export default MyProfile;
