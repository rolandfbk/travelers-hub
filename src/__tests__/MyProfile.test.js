import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../components/MyProfile/MyProfile';
import store from '../Redux/Store/ConfigureStore';

const MyProfileProvider = () => (
  <Provider store={store}>
    <MyProfile />
  </Provider>
);

it('renders correctly', () => {
  const tree = renderer.create(<MyProfileProvider />).toJSON();
  expect(tree).toMatchSnapshot();
});
