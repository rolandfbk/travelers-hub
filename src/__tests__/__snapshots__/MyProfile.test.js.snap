// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="my-profile"
>
  <div
    className="my-profile-align"
  >
    <div
      className="my-profile-item"
    >
      <h2>
        My Missions
      </h2>
      <p>
        Be the first to join...
      </p>
    </div>
    <div
      className="my-profile-item"
    >
      <h2>
        My Rockets
      </h2>
      <p>
        Be the first to reserve...
      </p>
    </div>
  </div>
</div>
`;
